src/components/Thanks/BadgesCustomization.vue

```diff
@@ -1,330 +1,63 @@
-							<div class="tw-mb-2">
-								<!-- eslint-disable-next-line max-len -->
-								<div
-									class="option-box hide-for-print"
-									:class="{'open' : openOrderConfirmation}"
-									@click="() => openOrderConfirmation = !openOrderConfirmation"
-									v-kv-track-event="[
-										'thanks',
-										'click',
-										'open-order-confirmation-drawer',
-									]"
-								>
-									<p class="tw-font-medium">
-										Show previous loan details
-									</p>
-									<kv-material-icon
-										:icon="mdiChevronDown"
-										class="expandable-button"
-										:class="{'tw-rotate-180' : openOrderConfirmation}"
-									/>
-								</div>
-								<kv-expandable
-									v-show="openOrderConfirmation"
-									easing="ease-in-out"
-								>
-									<div class="tw-py-2">
-										<checkout-receipt
-											v-if="receipt"
-											:lender="lender"
-											:receipt="receipt"
-										/>
-									</div>
-								</kv-expandable>
-							</div>
-							<div class="hide-for-print">
-								<div
-									class="option-box"
-									:class="{'open' : openShareModule}"
-									@click="() => openShareModule = !openShareModule"
-									v-kv-track-event="[
-										'thanks',
-										'click',
-										'open-share-drawer',
-									]"
-								>
-									<p class="tw-font-medium">
-										Share
-									</p>
-									<kv-material-icon
-										:icon="mdiChevronDown"
-										class="expandable-button"
-										:class="{'tw-rotate-180' : openShareModule}"
-									/>
-								</div>
-								<kv-expandable
-									v-show="openShareModule"
-									easing="ease-in-out"
-								>
-									<div class="tw-py-2">
-										<social-share-v2
-											v-if="receipt"
-											class="social-share"
-											:lender="lender"
-											:loans="loans"
-										/>
-									</div>
-								</kv-expandable>
-							</div>
-						</div>
-					</div>
-				</div>
-			</div>
-		</div>
-		<transition name="fade">
+		<div class="new-background tw-h-0 tw-w-0" :class="{ 'grow': newBgActive }"></div>
+		<transition-group name="fade" tag="div">
+			<FirstScreen
+				key="first-screen"
+				v-if="newScreenSteps === 0"
+				:lender="lender"
+				:loans="loans"
+				:is-guest="isGuest"
+				:selected-loan="selectedLoan"
+				:receipt="receipt"
+				@show-discover-badges="showDiscoverBadges"
+				@show-new-bg="showNewBg"
+			/>
+			<RevealedBadge
+				key="revealed-badge-screen"
+				@show-discover-badges="showDiscoverBadges"
+				v-if="newScreenSteps === 1"
+				:is-guest="isGuest"
+			/>
 			<DiscoverSection
-				v-if="discoverBadges"
+				v-if="newScreenSteps === 2"
+				key="discover-screen"
 				:selected-loan-region="selectedLoanRegion"
 				:is-guest="isGuest"
-				@back-to-earned-badge="() => discoverBadges = false"
+				:badges="randomSortedBadges"
+				@back="() => newScreenSteps -= 1"
+				@select-badge="selectBadge"
+			/>
+			<DetailSection
+				v-if="newScreenSteps === 3"
```

Comment:
`v-else-if` on these transition children. Vue 3 throws errors when it thinks there could be more than one child of a transition.